modelforge/potential/potential.py

```diff
@@ -305,6 +305,63 @@ def _remove_pairlist(self, processed_output: Dict[str, torch.Tensor]):
         del processed_output["r_ij"]
         return processed_output
 
+    @torch.jit.export
+    def set_neighborlist_strategy(self, strategy: str, skin: float = 0.1):
+        """
+        Set the neighborlist strategy and skin for the neighborlist module.
+        Note this cannot be called from a JIT-compiled model.
+
+        Parameters
+        ----------
+        strategy : str
+            The neighborlist strategy to use.
+        skin : Optional[float], optional
+            The skin for the Verlet neighborlist (default is None).
+        """
+        self.neighborlist._set_strategy(strategy, skin=skin)
+        # self.neighborlist.strategy = strategy
+        # self.neighborlist.skin = skin
+        # self.neighborlist.skin = skin
+        # self.neighborlist.half_skin = skin * 0.5
+        # self.neighborlist.cutoff_plus_skin = self.neighborlist.cutoff + skin
+
+    @torch.jit.export
+    def forward_for_jit_inference(
```

Comment:
can you clarify what this function does?

Reply:
I will add some comments to the code and add it to the docstrings as well.  Basically, I just created a separate forward function because I couldn't pass the NNPInput class directly (as discussed above, it sees NNPInput as a different class because it was compiled outside...basically torch script does not like accepting class functions; it wants tensors or basic types like dicts of tensors).  This function allows us to just pass the individual values when calling from the openmm wrapper.   This was the simple work around because I think NNPInput is very useful and should still be used.

Reply:
Thank you for clarfiying! That's a good rational!